src/main/java/com/internet/shop/controller/user/GetUserDetailsAdminController.java

```diff
@@ -0,0 +1,30 @@
+package com.internet.shop.controller.user;
+
+import static com.internet.shop.controller.LoginController.USER_ID;
+
+import com.internet.shop.library.Injector;
+import com.internet.shop.service.interfaces.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetUserDetailsAdminController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final UserService userService =
+            (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long userId;
+        try {
+            userId = Long.parseLong(req.getParameter(USER_ID));
+        } catch (NumberFormatException exception) {
+            userId = (Long) req.getSession().getAttribute(USER_ID);
+        }
```

Comment:
in which case does this work?

Reply:
When we want to get the certain user's details from the All Users list

Reply:
In this case we send the user's id from the *.jsp
If the logged in common user clicks on the Account Details button at the navigation bar we extract his id from the session

Reply:
<img width="1552" alt="Screenshot 2020-09-17 at 12 22 28" src="https://user-images.githubusercontent.com/68664175/93452013-92f34280-f8e0-11ea-93de-8a11d779d09e.png">
<img width="1552" alt="Screenshot 2020-09-17 at 12 22 44" src="https://user-images.githubusercontent.com/68664175/93452025-98e92380-f8e0-11ea-997b-718c88d6bff0.png">
<img width="1552" alt="Screenshot 2020-09-17 at 12 22 23" src="https://user-images.githubusercontent.com/68664175/93452030-9a1a5080-f8e0-11ea-8e68-81d4e7bebf72.png">

Reply:
I've got two different controllers for admin and user, sorry
Removing this :D